Jenkinsfile

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env groovy
+
+import groovy.transform.Field
+
+@Field
+def TAG = ""
```

Comment:
Might be due to lack of my pipeline understanding but why can't this be a toplevel `environment` directive instead?